src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,44 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
It isn't good practice to make long methods, especially public ones.
Please split your solution to three private methods like readFile, generateReport and writeToFile.
Then in your public method make only calls to there 3 methods to make it short and clear